docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,234 @@
+---
+id: googledrive
+title: Google Drive Connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive Connector** allows you to create empty folders or documents on [Google Drive](https://drive.google.com/) from templates from your BPMN process.
+
+## Prerequisites
+
+To start working with the **Google Drive Connector**, a relevant OAuth token must be configured and stored as a secret in the user's cluster. The token must have permission to read and create a folder and/or files from a desired Google Drive instance. Follow the steps from the [appendix](#appendix--faq) to find out more about creating an OAuth token and giving relevant permissions.
+
+## Create a Google Drive Connector task
+
+Currently, the **Google Drive Connector** supports two types of templates: create a folder and create a file from a template.
+
+To use a **Google Drive Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](../use-connectors.md) to learn more.
+
+## Make your Google Drive Connector for creating a folder executable
```

Comment:
To remove redundancy and get away from this long heading, have you considered a structure like this:

```
## Make your Google Drive Connector executable

To make the **Google Drive Connector** executable, fill out the mandatory fields highlighted in red in the properties panel:

![Google Drive Connector new folder example](../img/connectors-googledrive-new-folder-filled.png)

For all oeprations, you need to fill the required credentials in the **Authentication** section. See the [relevant appendix entry](#how-can-i-authenticate-my-connector) to find out more.

You can then decide between two operations: [creating a folder](link) or [creating a file from template](link)

### Create a new folder
...

### Create a file from a template
...
```

Reply:
Agreed. Updated.